sympy/utilities/lambdify.py

```diff
@@ -178,6 +178,102 @@ def _import(module, reload=False):
 # linecache.
 _lambdify_generated_counter = 1
 
+def pre_treatment_cse(args_f, expr):
+    r'''
+    This function does a pretreatment of the expression to prevent errors when
+    putting a Derivative as an argument to lambdify. The Derivatives present in
+    the arguments that also appear in the function are replaced by various strings.
+    This function returns the new expressions with the instances of Derivatives replaced
+    and a dictionnary that contains the associations of changes
+    (ex : Derivative(x, u) : "x0")
+    '''
+    #Necessary librairies and dependencies
+    from sympy.core.function import Derivative
+    from sympy.core.symbol import Symbol
+    from sympy.core import Basic
+    from sympy.matrices.expressions import MatrixSymbol
+    from sympy.matrices.expressions.matexpr import MatrixElement
+    from sympy.polys.rootoftools import RootOf
+    from sympy.utilities.iterables import numbered_symbols, iterable
+    #creation of the dictionnary
+    dictionnary={}
+    # creation of the symbols that can't be used to replace the Derivatives in the expression
+    excluded_symbols = set()
+    symbols = numbered_symbols(cls=Symbol)
+    def _eliminates_symbols(expr) :
```

Comment:
```suggestion
    def _eliminates_symbols(expr):
```

Reply:
yes, you're right, sorry I must have put it there by mistake, thank you!